cdc/sink/dmlsink/mq/worker.go

```diff
@@ -295,6 +304,12 @@ func (w *worker) sendMessages(ctx context.Context) error {
 				return errors.Trace(err)
 			}
 			for _, message := range future.Messages {
+				if message.ClaimCheckFileName != "" {
```

Comment:
why not check the message size here ?

Reply:
it's the encoder's responsibility to check the size, if the size is larger than the limit, it will set the `ClaimCheckName`

Reply:
Add comments explaining the meaning when the value of the message.ClaimCheckFileName  is empty.